src/components/chat/chatBot/index.tsx

```diff
@@ -49,18 +49,26 @@ const ChatBot: React.FC = () => {
     setIsOpen(!isOpen)
   }
 
-  useMemo(() => {
+  useEffect(() => {
     if (!isOpen) return
     if (!profileID) return
-    ws.current = new WebSocket(`ws://localhost:3030?agentID=${profileID}&secondaryID=copilot`)
+    const WEBSOCKET_ENDPOINT = process.env.NEXT_PUBLIC_WEBSOCKET_ENDPOINT
+    ws.current = new WebSocket(`ws://${WEBSOCKET_ENDPOINT}?agentID=${profileID}&secondaryID=copilot`)
     ws.current.onopen = () => {
       console.log('Connected to server')
     }
     ws.current.onmessage = (event) => {
-      // console.log('Received message')
+      console.log('Received message:', event.data)
```

Comment:
Do we need to log? Para al menos limpiar la consola lo más posible, no sé si lo vayan a considerar malas prácticas.